### DISCUSSION TITLE
Have tried everything and can't get the email confirmations to work.

### DISCUSSION THREAD

**Discussion by Parminio** [mastodon/mastodon#17923] [Q&A] [answered]
I've been reading different pages all day long trying to solve it but nothing has helped.

In Sidekiq, this is what I see no matter what I do:

![sidekiq shot](https://user-images.githubusercontent.com/88104702/161102726-8a6795da-1b32-4340-aafb-2ba6e08ed4d2.jpg)

These are the settings:

![env production settings](https://user-images.githubusercontent.com/88104702/161102837-2035141d-418a-4263-8ef8-d983d025423b.jpg)

I have been trying different things for hours on end. I even tried signing up for a free SMTP account on SendGrid and that didn't work either.

Any help would be great.
Thanks in advance.

**Comment by Gargron** [comment, marked as answer]
Some VPS providers have a default outgoing firewall rule blocking SMTP connections (I remember this in Scaleway). Could be that.

I don't know if your SMTP settings are correct since it's provider-specific.

**Comment by Parminio** [comment]
Went with Sendinblue free SMTP service and it works like a charm. Kind of crappy that the domain and web service I pay for doesn't work but one for free does.

Oh well.

Thanks for the input.
